template/model-server/catalog-info.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: {{ values.modelServerName }}
+  annotations:
+    backstage.io/techdocs-ref: dir:./
+  description: '{{ values.modelServerDescription }}'
+  links:
+    - url: {{ values.modelServerURL }}
+      title: Access
+      type: website
+      icon: WebAssett
+    - url: {{ values.modelServerAPIURL }}
+      title: API URL
+      type: website
+      icon: WebAsset
+  tags: {{ values.modelServerTags }}
+spec:
+  type: model-server
+  owner: user:{{ values.modelServerOwner }}
+  lifecycle: production
+  {%- if values.modelServerAPI %}
+  providesApis:
+    - {{ values.modelServerAPIName }}
+  dependsOn:
+    - api:{{ values.modelServerAPIName }}
+  {%- endif %}
+  profile:
+    displayName: {{ values.modelServerDisplayName }}
+---
+{%- if values.modelServerAPI %}
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: {{ values.modelServerAPIName }}
+  description: '{{ values.modelServerAPIDescription }}'
+  tags: {{ values.modelServerAPITags }}
```

Comment:
this bleeds into the example repo, and overall model catalog format, but during our chat with @amfred, you @johnmcollier and I we talked about adding the links to the API object as well

should we start introducing that change with this PR, or do you want to update the example repo, then come back and update this?

Reply:
Yeah, let's do that. The example model catalog that our team uses will eventually be generated from this template, so we should make that change here

Reply:
Yeah, agreed, we should make the change here.

Since we'll eventually be generating our reference model catalogs from this template, this should be where we make the change, and I think now is a good time to do so.

Updated.